source/community/release_notes/quince.rst

```diff
@@ -27,14 +31,36 @@ Instructor Experiences
 Administrators & Operators
 **************************
 
+* `feat: feature flag to disable Advanced Settings <https://github.com/openedx/edx-platform/pull/32015>`__
+
+Which MFEs will be included?
+
+* `frontend-app-learner-record: <https://github.com/openedx/frontend-app-learner-record>`__
+* `frontend-app-learner-dashboard: <https://github.com/openedx/frontend-app-learner-dashboard>`__
```

Comment:
These are both user-facing features. I think we included some information about the learner dashboard in the palm release that you could copy and paste. A screenshot would be helpful, too. 

The learner record MFE was added provisionally in olive, I think. I don't recall if there was much in the release notes about it but there might be something you can copy from there.

Reply:
I can't find the references you cites. We can handle this in a subsequent revision.